### PR TITLE
Add set/get power channel functions

### PR DESCRIPTION
**Pull request by ArashYzd**
These functions have been tested with the EPS. They allow setting/getting the status of the power channels.

### CODE REVIEW

File: hardware_interface/source/eps.c [hunk 1]
```diff
@@ -141,16 +141,31 @@ void EPS_getHK(eps_instantaneous_telemetry_t* telembuf) {
...
+/* Sends a command to eps to set the status of a power channel */
+void eps_set_pwr_chnl(uint8_t pwr_chnl_port, bool status){
+    int8_t response[2];
+    uint8_t cmd[5] = {0};
+    cmd[0] = 0; //single output control
+    cmd[1] = pwr_chnl_port;
+    cmd[2] = status;
+    // delay = 0 so cmd{4] = cmd[5] = 0
+    csp_transaction_w_opts(CSP_PRIO_LOW, EPS_APP_ID, EPS_POWER_CONTROL,
+                           10000, &cmd, sizeof(cmd), &response,
```
Comment (arrooney) [inline hardware_interface/source/eps.c:167]: Are we just ignoring the reponse? What is the response? If we don't want it we should use `csp_sendto`

**Review by arrooney — CHANGES_REQUESTED** [review]

File: hardware_interface/source/eps.c (hunk 1 above)
Comment (ArashYzd) [reply hardware_interface/source/eps.c:167]: I guess the response could become useful at some point. I'll add a return value.

**Comment by ArashYzd** [comment]
The comment has been addressed.

File: hardware_interface/source/eps.c [hunk 2]
```diff
@@ -141,16 +141,32 @@ void EPS_getHK(eps_instantaneous_telemetry_t* telembuf) {
...
 
-int8_t eps_get_pwr_chnl(uint8_t pwr_chnl_port){
-
+/* Gets the status of the power channel */
+uint8_t eps_get_pwr_chnl(uint8_t pwr_chnl_port){
+    eps_refresh_instantaneous_telemetry();
+    eps_instantaneous_telemetry_t eps = get_eps_instantaneous_telemetry();
+    uint32_t outputStatus = eps.outputStatus; // a codeword that has the status of all channels
+    uint8_t pwr_chnl_status = (uint8_t) (outputStatus >> (pwr_chnl_port - 1)) & 1; // chnl_port : 1-18
+    return pwr_chnl_status;
```
Comment (QuocTrungTran) [inline hardware_interface/source/eps.c:155]: There is no problem here. But if I were you, I just do `return (uint8_t) (eps.outputStatus >> (pwr_chnl_port -1)) & 1;`  😆

**Review by QuocTrungTran — APPROVED** [review]
lgtm!

File: hardware_interface/source/eps.c (hunk 2 above)
Comment (ArashYzd) [reply hardware_interface/source/eps.c:155]: Thanks. This way it's more obvious what the function is returning.

**Review by arrooney — APPROVED** [review]
Could you add a task to write unit tests for this with a mocked get_eps_instantaneous_telemetry

**Comment by ArashYzd** [comment]
> Could you add a task to write unit tests for this with a mocked get_eps_instantaneous_telemetry

Sure. I'll make a task for both this and the GS test script for EPS commands.

**Pull request merged**